src/diffusers/pipelines/animatediff/pipeline_animatediff_video2video.py

```diff
@@ -641,9 +619,7 @@ def prepare_latents(
         if video and not isinstance(video[0], list):
             video = [video]
         if latents is None:
-            video = torch.cat(
-                [self.image_processor.preprocess(vid, height=height, width=width).unsqueeze(0) for vid in video], dim=0
-            )
+            video = self.video_processor.preprocess_video(video, height=height, width=width, preceed_with_frames=True)
```

Comment:
can we move this outside of the `prepare_latents`?
let's do something similar totext_to_video img2img

Reply:
basically like this https://github.com/huggingface/diffusers/blob/0d23645bd120e6785a3d81ee8e053afe84bbc77f/src/diffusers/pipelines/text_to_video_synthesis/pipeline_text_to_video_synth_img2img.py#L690
but use the `video_processor `